src/transformers/onnx/config.py

```diff
@@ -297,6 +314,12 @@ def generate_dummy_inputs(
                 The width of the generated images.
             image_height (`int`, *optional*, defaults to 40):
                 The height of the generated images.
+            sampling_rate (`int`, *optional* defaults to 22050)
+                The sampling rate for audio data generation.
+            time_duration (`int`, *optional* defaults to 5 sec)
```

Comment:
```suggestion
            time_duration (`float`, *optional* defaults to 5.0)
```
Let's be consistent with the signature!

Reply:
Done